src/assertError.spec.ts

```diff
@@ -0,0 +1,19 @@
+import test from 'ava'
+import AssertError from './assertError'
+
+test('test error format based on input', _t => {
```

Comment:
When you are using the variable, you don't have to prefix it with `_`.
That is a convention to indicate a parameter is not being used.